.github/workflows/sims.yml

```diff
@@ -34,7 +34,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v2.1.4
         with:
-          go-version: 1.15
+          go-version: 1.16
```

Comment:
Why is this 1.16?

Reply:
this was fixed in 9fc9028b51a645ec4c1857caebeb3e7d1d9334b5 , thanks for finding it though, you're quick!